pom.xml

```diff
@@ -12,8 +12,7 @@
   <packaging>hpi</packaging>
 
   <properties>
-    <jenkins.version>2.277.4</jenkins.version>
-    <java.level>8</java.level>
```

Comment:
also delete https://github.com/jenkinsci/sse-gateway-plugin/blob/c8f9b702b1d0e7847cb8c3b437de501586b1c34e/pom.xml#L118-L128 for clarity